src/encoded/tests/datafixtures.py

```diff
@@ -29,7 +29,8 @@
     'workflow_mapping', 'workflow_run_sbg', 'workflow_run_awsem',
     'tracking_item', 'quality_metric_flag',
     'summary_statistic', 'summary_statistic_hi_c', 'workflow_run',
-    'microscope_configuration', 'image_setting', 'quality_metric_mcool'
+    'microscope_configuration', 'image_setting', 'quality_metric_mcool',
+    'ingestion_submission', 'file_other'
```

Comment:
@aschroed and @willronchetti Isn't this list supposed to be topologically sorted? Why do we lay it out by hand? How do we know it's in the right order? I'm guessing this should be earlier in the list, though by accident it might actually not matter since I assume this ordering is a tangle of partial orders in disguise.

Reply:
The order of items in the list is relevant and is based on dependencies to other items.  For example it would be problematic to have 'file_other' before 'file_format' as the file other insert to load will depend on being able to find the relevant 'file_format' item.  As long as no other item type depends on having 'ingestion_submission', 'file_other' inserts present in the db before these inserts are loaded for tests (and I don't think there are but have not looked to be sure) then it should be fine to have them at the end.